README.md

```diff
@@ -805,10 +805,52 @@ method. It takes 3 arguments: the first is the name of the section as
 referred to on the page (sections that appear on multiple pages can be
 named differently). The second argument is the class of which an
 instance will be created to represent the page section, and the third
-argument is a css selector that identifies the root node of the section
+argument the search argument that identifies the root node of the section
 on this page (note that the css selector can be different for different
 pages as the whole point of sections is that they can appear in
-different places on different pages).
+different places on different pages). It can also take the block with
+defenition of section. This allows to avoid creating separate section
+class (just for scoping) or then you defining sections collection
+(used only once).
+
+```ruby
+class SectionExperiments < SitePrism::Page
+  sections :level_1, '.level-1' do
+    element :element1, '.element-1'
+  end
+end
+```
+
+Providing class and block allows also to extend section for this
```

Comment:
This can be rewritten to something like ...
```
If you define a section as a class and as an Anonymous section, you can have some handy constructs like the one below

<insert ruby code>
```

I'd also move this advice to below here: https://github.com/natritmeyer/site_prism/pull/250/files#diff-04c6e90faac2675aa89e2176d2eec7d8R1180